pkg/kubectl/cmd/run.go

```diff
@@ -66,10 +70,13 @@ func NewCmdRun(f *cmdutil.Factory, out io.Writer) *cobra.Command {
 	cmd.Flags().Int("port", -1, "The port that this container exposes.")
 	cmd.Flags().Int("hostport", -1, "The host port mapping for the container port. To demonstrate a single-machine container.")
 	cmd.Flags().StringP("labels", "l", "", "Labels to apply to the pod(s).")
+	cmd.Flags().BoolP("interactive", "i", false, "Keep stdin open on the container(s) in the pod, even if nothing is attached.")
+	cmd.Flags().Bool("tty", false, "Allocated a TTY for each container in the pod.")
```

Comment:
exec and attach using "t" as the short alias. We should do that everywhere or nowhere. 

Does run support `-o template`?